data/remnant/remnant.txt

```diff
@@ -276,6 +276,490 @@ phrase "remnant"
+	word
+		"I am"
+		"We are"
+	word
+		" "
+	word
+		"watching"
+		"observing"
+		"analyzing"
+		"overseeing"
+	word
+		" "
+	word
+		"you"
+		"your actions"
+		"your behavior"
+	word
+		"."
+
+phrase "remnant uncontacted hostile"
+	word
+		"Eat space dust"
+		"Time to go"
+		"Sing your last prayers"
+		"Leave this space"
+	word
+		","
+	word
+		" "
+	word
+		"hostile ship"
+		"pirate"
+		"Alpha"
+	word
+		"!"
+
+phrase "remnant uncontacted hostile"
+	word
+		"I"
+		"My friend"
+		"My comrade"
+		"My gut"
+	word
+		" "
+	word
+		"was right"
+		"was correct"
+		"knew it"
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"You're"
+	word
+		" "
+	word
+		"hostile"
+		"here with malicious intent"
+		"an Alpha"
+	word
+		"!"
+
+phrase "remnant uncontacted hostile"
+	word
+		"(They sent you hail consisting of a sign language you don't understand, and it doesn't seem friendly.)"
+		"(They sign so quickly that your hail receiver can't process it all.)"
+		"(It seems you are not welcome here.)"
+
+phrase "friendly remnant"
+	word
+		"Greetings"
+		"Howdy"
+		"Hello there"
+		"Salutations"
+	word
+		", "
+		" "
+	word
+		"human"
+		"friend"
+		"comrade"
+		"brethen"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"I almost died"
+		"Several hostiles attacked my ship"
+		"My mission was difficult"
+		"We had an emergency"
+		"There was a swarm of Korath raiders here"
+		"We faced a life-threatening situation"
+	word
+		" "
+	word
+		"last time"
+		"last week"
+		"last month"
+	word
+		"."
+		"!"
+	word
+		"Luckily"
+		"Fortunately"
+		"Miraculously"
+		"Suddenly"
+	word
+		","
+	word
+		" "
+	word
+		"I"
+		"my captain"
+		"we"
+	word
+		"actually had"
+		"remembered the ship had"
+	word
+		"a cloaking device all along"
+		"a secret weapon the whole time"
+		"a last resort every Remnant should have"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"If"
+	word
+		" "
+	word
+		"you have"
+		"the prefects gave you"
+		"our elders allowed you"
+		"our government granted you"
+	word
+		" "
+	word
+		"a license"
+		"a permit"
+		"permission to buy our outfits"
+	word
+		","
+	word
+		" "
+	word
+		"I"
+		"our experts"
+		"our veterans"
+		"my friends"
+	word
+		" "
+	word
+		"would suggest"
+		"would recommend"
+	word
+		" "
+	word
+		"buying"
+		"getting"
+	word
+		" "
+	phrase
+		"remnant inhibitor cannon"
+		"remnant thrasher cannon"
+		"remnant emp torpedo bay"
+		"remnant salvage scanner"
+	word
+		"They're the best!"
+		"You should definitely try it!"
+		"Definitely better than those outfits in your space."
+
+phrase "remnant inhibitor cannon"
+	word
+		"some"
+		"those"
+		"our"
+	word
+		" "
+	word
+		"inhibitor cannons."
+	word
+		" "
+	word
+		"They"
+		"Those weapons"
+	word
+		"can"
+		"will"
+	word
+		" "
+	word
+		"disrupt"
+		"disable"
+		"slow"
+	word
+		" "
+	word
+		"the enemy's thrusters"
+		"your target's steering"
+		"any engine"
+	word
+		" "
+	word
+		"and prevent them from escaping"
+		", and anyone won't stand a chance"
```

Comment:
This is an odd phrasing to me. Why have such a broad "anyone" as the subject when the previous words give narrow subjects?
```suggestion
		"and they won't stand a chance"
```